### DISCUSSION TITLE
책 독후감 관련

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#32] [Q&A] [answered]
블로그에 '북 리뷰'카테고리를 만들고 저번에 받은 책 내용이나 앞으로 개발에 관련된 책 리뷰를 남기고 싶은데 독후감을 쓴지 너무 오래되기도 했고 개발 관련 책은 어떤 방식으로 글을 남겨야 하는지 잘 모르겠습니다.  

개발책 리뷰에 필수로 들어가면 좋다는 내용이나 형식을 알려주실 수 있나요..?

**Comment by jongfeel** [comment, marked as answer]
어떤 형식에 맞춰서 쓰는 것 보다는 자연스럽게 생각한 걸 정리한다는 느낌으로 해보시면 좋을 것 같네요.

팁을 하나 드리면
책 내용 정리하는 거 100%로 하기 보다는
자신의 생각, 의견, 개선점 등을 추가하는게 좋긴 합니다.
